awx/ui/src/screens/InstanceGroup/Instances/InstanceList.js

```diff
@@ -224,13 +224,15 @@ function InstanceList() {
                   ]
                 : []),
               <DisassociateButton
-                verifyCannotDisassociate={selected.some(
-                  (s) => s.node_type === 'control'
-                )}
+                verifyCannotDisassociate={
+                  selected.some((s) => s.node_type === 'control') ||
+                  instanceGroup.name === 'controlplane'
```

Comment:
We only need to prevent disassociation of an hybrid instance if the instance group it belongs to is a protected instance group.  IIRC my thought process was to put that logic here so as to help keep the logic in the instancegroup/instances/instancelist from being too complicated.

Reply:
Thanks for clarifying @AlexSCorey